### PR TITLE
Update development status to beta

### PR DESCRIPTION
**Pull request by adamchainz**
It looks like trycast is pretty mature and well tested, so upgrade its status to "stable" to reassure potential users (at least, me).

### CODE REVIEW

**Comment by rednafi** [comment]
I second to this as well. Such a nifty little tool.

**Comment by davidfstr** [comment]
I'm going through a rough patch in my personal life right now, so I will respond briefly.

I'd be okay with upgrading the status of this project to "beta", but I feel like "stable" would require some additional documentation. The [roadmap](https://github.com/davidfstr/trycast/wiki/Roadmap) is suggestive of my thoughts here.

Right now there is a quickstart but no reference or API documentation to speak of.

File: pyproject.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ homepage = "https://github.com/davidfstr/trycast"
 repository = "https://github.com/davidfstr/trycast"
 #documentation = "TODO"
 classifiers = [  # https://pypi.org/classifiers/
-    "Development Status :: 3 - Alpha",
+    "Development Status :: 5 - Production/Stable",
```
Comment (adamchainz) [inline pyproject.toml:12]:
```suggestion
    "Development Status :: 4 - Beta",
```

**Comment by adamchainz** [comment]
Thanks for the update @davidfstr 

I've modified this PR to change to "beta" then.

**Comment by rednafi** [comment]
Yeah beta is much better. At least, it gives me more confidence in trying it out. Thanks!

**Pull request merged**